virt/virt-2-6-release-notes.adoc

```diff
@@ -95,6 +97,34 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
+* The Containerized Data Importer (CDI) and KubeVirt depend on QEMU which does not support NFS version 3. Therefore, only NFS version 4 is supported. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1892445[*BZ#1892445*])
+
+* The name of the Fedora PVC in the `openshift-virtualization-os-images` namespace is `fedora`, instead of `fedora32`. If you populated the `fedora32` PVC in {VirtProductname} 2.5 or earlier, the virtual machine does not appear in the web console and you cannot use it to clone another virtual machine. As a workaround, upload a Fedora image by naming the PVC `fedora`, instead of `fedora32`. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913352[*BZ#1913352*])
+
+* When creating a HPP boot source, the data volume is `pending` with a `WaitForFirstConsumer` status if a user creates the boot source using any method except the *Upload local file (creates PVC)* option. To resolve this issue, edit the YAML that defines the PVC in the *Storage* > *Persistent Volume Claims* screen.
++
+** As a workaround, edit the YAML of the underlying PVC of the data volume to add the `cdi.kubevirt.io/storage.bind.immediate.requested: "true"` `annotation`:
++
+[source,yaml]
+----
+metadata:
+annotations: cdi.kubevirt.io/storage.bind.immediate.requested: "true"
+----
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1929177[*BZ#1929177*])
+
+* If you use a Fedora image as a boot source, it is no longer attached to a template if the PVC that you used to attach the boot source was previously provisioned. A new Fedora PVC with the name `fedora` should be attached to a template before you can use the template to create virtual machines from boot sources. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1907187[*BZ#1907187*]) (link:https://bugzilla.redhat.com/show_bug.cgi?id=1913352[*BZ#1913352*])
+
+//The issues below are from the 2.5 release. See items above for new known issues for 2.6.
+//The below issues have not yet been pruned/updated for 2.6.
+
+* When a node fails in user-provisioned installations of {product-title} on bare metal deployments, the virtual machine does not automatically restart on another node. Automatic restart is supported only for installer-provisioned installations that have machine health checks enabled. Learn more about xref:/../virt/install/preparing-cluster-for-virt#preparing-cluster-for-virt[configuring your cluster for {VirtProductName}].
+
+* If your {product-title} cluster uses OVN-Kubernetes as the default Container Network Interface (CNI) provider, you cannot attach a Linux bridge or bonding to the default interface of a host because of a change in the host network topology of OVN-Kubernetes. As a workaround, you can use a secondary network interface connected to your host or switch to the OpenShift SDN default CNI provider. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1887456[*BZ#1887456*])
```

Comment:
Not sure if these are your changes , but this known issue appears twice in the list. I am aware that the instance below references the correct BZ, so you can probably delete this.

Reply:
Deleted 1887456